content/en/sensitive_data_scanner/_index.md

```diff
@@ -119,28 +172,91 @@ You can create custom scanning rules using regex patterns to scan for sensitive
 {{% sds-scanning-rule %}}
 1. Click **Add Rule**.
 
+**Notes**:
```

Comment:
🛑 Notes should be outside of "Add a custom scanning rule" collapsible section.

Reply:
Resolved